lib/registrant.rb

```diff
@@ -0,0 +1,56 @@
+class Registrant
+
+    attr_reader :name, 
+                :age, 
+                :permit, 
+                :license_data
+
+    def initialize(name, age, permit = false, license_data = {})
```

Comment:
No need to have license_data as a 4th argument here. We just want it in the initialize like you have it on line 12 which will set the hash values for a new Registrant.